infrastructure/baseline.tf

```diff
@@ -15,10 +15,7 @@ resource "azurerm_resource_group" "rg_project" {
 }
 
 locals {
-  # Get the primary region from the regions map:
-  primary_region = {
-    for region_key, region in var.regions :
-  region_key => region if region.is_primary_region }
+  primary_region = [for k,v in var.regions : k if v.is_primary_region][0]
```

Comment:
I suspect if you did terraform fmt that the space between k,v would be added. So this suggests to me that you have not done terraform fmt. Can you run it please?

Reply:
Done